examples/emodel_pipeline_local_python/final.json

```diff
@@ -1,69 +1,692 @@
 {
```

Comment:
Do we need to upload the final.json? This is not needed to run the pipeline, it is an output of the pipeline.